mappings/net/minecraft/util/dynamic/Codecs.mapping

```diff
@@ -167,7 +167,7 @@ CLASS net/minecraft/unmapped/C_tyjpezxh net/minecraft/util/dynamic/Codecs
 	METHOD m_mcwzjvkz (Ljava/util/List;)Lcom/mojang/serialization/DataResult;
 		ARG 0 list
 	METHOD m_mfpnjepb (Lorg/joml/Vector3f;)Ljava/util/List;
-		ARG 0 vector3f
+		ARG 0 vector3
```

Comment:
```suggestion
		ARG 0 vector
```
I don't like `vector3` here because 1) it's half a suffix and 2), it makes it look like it means 'the third vector' (especially next to `list2`).
If just `vector` isn't favored I would prefer it go back to `vector3f`.